### PR TITLE
Destination card icons

### PR DESCRIPTION
**Pull request by infamouskeyduster**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Migration 
- [X] Styling
- [ ] Testing

### Commit Emojis:
:tada: Initial commit
:construction: HTML modifications/updates.
:art: CSS modifications/updates.
:crystal_ball: JS modifications/updates.
:lipstick: hotfix any code.
:bar_chart: new tests
:camera_flash: adding screenshots
:bento: Adding new assets or images
:beetle: If it is a bug or issue fix, describe the issue that was fixed.

### Detailed Description:
This adds small non-breaking changes that incorporate an icon into the destination card, as well addresses issues with the homepage div and animation speeds. 

### Files modified:
See PR request.

### CODE REVIEW

File: src/Components/App/App.css [hunk 1]
```diff
@@ -13,12 +13,15 @@
...
   /* background-color: black; */
 }
 
 .background {
   /* margin-top: 10%; */
   z-index: -1;
   position: absolute;
+  /* width: 100vw;
+  height: 100vh; */
+  /* background-repeat: none; */
```
Comment (lilianaweimer) [inline src/Components/App/App.css:26]: We can probably remove these in the future!

Comment (infamouskeyduster) [reply src/Components/App/App.css:26]: Good call! I will remove them now!

File: src/Components/Background/Background.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
-    <>
-    <div id="stars"></div>
-    <div id="stars2"></div>
-    <div id="stars3"></div>
-    </>
-  )
-}
-
-export default Background
+class Background extends PureComponent {
```
Comment (lilianaweimer) [inline src/Components/Background/Background.js:4]: We might be able to safely change this back to a functional component

**Review by lilianaweimer — APPROVED** [review]
Looks great! So glad we figured out the weird bug with the background 🤘

**Review by LeahWinters — APPROVED** [review]
Great job team getting those little bugs kicked out right away! Loving how this is looking so far!

**Comment by LeahWinters** [comment]
Thanks for going in real quick to clean up the code before we merged!

**Pull request merged**